irr/include/mt_opengl.h

```diff
@@ -588,6 +588,7 @@ class OpenGLProcedures final {
 	typedef void (APIENTRYP PFNGLGETOBJECTLABELPROC_MT) (GLenum identifier, GLuint name, GLsizei bufSize, GLsizei * length, GLchar * label);
 	typedef void (APIENTRYP PFNGLOBJECTPTRLABELPROC_MT) (const void * ptr, GLsizei length, const GLchar * label);
 	typedef void (APIENTRYP PFNGLGETOBJECTPTRLABELPROC_MT) (const void * ptr, GLsizei bufSize, GLsizei * length, GLchar * label);
+	typedef void (APIENTRYP PFNGLOBJECTLABELKHRPROC_MT) (GLenum type, GLuint object, GLsizei length, const GLchar * label);
```

Comment:
this file is automatically generated, we can't edit it

Reply:
I changed generatuon script to use:
`ParseHeader( glHeaderPath .. "/gl2ext.h", procedures, funcRegexES, List(), consts, nameset );`
instead of
`ParseHeader( glHeaderPath .. "/gl2ext.h", procedures, funcRegexES, List(), consts, nameset, true );`

And it generates:
```
  if (!ObjectLabel) ObjectLabel = (PFNGLOBJECTLABELPROC_MT)cmgr->getProcAddress("glObjectLabel");
  if (!ObjectLabel) ObjectLabel = (PFNGLOBJECTLABELPROC_MT)cmgr->getProcAddress("glObjectLabelKHR");
```

This still results in using glObjectLabel instead of glObjectLabelKHR for OpenGL ES less than 3.2.

Unfortunately, it looks like a non-NULL pointer is not forbidden to be returned to a function supported by higher OpenGL ES.

Reply:
Hmm, that's tricky.
I suppose we could just diable the KHRDebug support for GLES to fix this situation.

Reply:
So I made a few changes to the generation script. Output should be now compatible with GL ES and it is on the user to select the correct function to call, based on the OpenGL or OpenGLES version and available extensions.